### DISCUSSION TITLE
Please fix BG's/WSG to scale based on time played not flags capped.

### DISCUSSION THREAD

**Discussion by QuiexVZ** [chromiecraft/chromiecraft#7301] [Suggestions]
The idea - 
The BG needs a TIMER system for honor gain. Not flags. People who put in the time to show up at BG's win or lose should be able to walk away with something decent for their time even if they're wrecked. The value of flag caps need to be reduced to 20% of the total BG honor gained at best. If PvP BG's can't be equalized and adjusted for fresh people than it should at least be reasonably profitable for the time spent doing them. (Please note I speak only to BG's not Arena. Arena is a whole different thing.) 

The rant -
I like BG, honor serves as a quick and easy way to get certain BiS gear pieces. It serves as a bridge between between a once a week kara which MAY get you gear, and grinding every single heroic every single day to get you SOME pieces. With Pugs already calling for certain gear scores to raid there needs to be a way to fill that gap. 

Enter the bullies. They don't give a royal @#$@# about anyone but themselves. It's such a dichotomy. This server is so friendly in many respects, but if it comes to pvp you get hardcore dirt bags (other words I could use). I'm a bear tank, im cut down in 3 hits and graveyard farmed becuase "they can". What to I get to show for it when they win? Garbage honor. The strong get stronger and the weak get screwed (no, i wouldn't do that in their place with how the system is set up now. I take deep offence to that. I am NOT a bully).

**Comment by Never6** [comment]
honor prices should be reduced as s1 pieces cost way to much considering how small the server is
and a lot of them flat out dont make any sense
why does idol cost the same price as a 2h weapon????
